spec/components/notification_component_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe NotificationComponent, type: :component do
+  let(:content) { 'This is content' }
+  let(:style) { 'notice' }
+  let(:dismiss) { true }
+  let(:background) { false }
+  let(:alert) { false }
+  let!(:inline_component) { render_inline(NotificationComponent.new(content: content, style: style, dismiss: dismiss, background: background, alert: alert)) }
+
+  describe '#content' do
+    context 'when content is a string' do
+      it 'renders content in the body' do
+        expect(inline_component.css('.govuk-notification__body').to_html).to include(content)
```

Comment:
I would prefer to use `'This is content'` over `content` as it makes the test more robust (there's a small chance of a bug in the test making `content` evaluate to `nil`, for example). Up to you though.

Reply:
Done!